felix/bpf/bpf.go

```diff
@@ -78,7 +78,7 @@ const (
 	sockmapEndpointsMapName    = "calico_sk_endpoints_" + sockmapEndpointsMapVersion
 
 	DefaultBPFfsPath = "/sys/fs/bpf"
-	CgroupV2Path     = "/run/calico/cgroup"
+	CgroupV2Path     = "/var/run/calico/cgroup"
```

Comment:
`/var/run` is a symlink to `/var/` so these two path are identical, but thought to change it to prevent confusion.